core/sections/clause_2_conformance.adoc

```diff
@@ -54,11 +56,12 @@ Additional capabilities such as support for transactions, extended job monitorin
 
 Conformance with this standard SHALL be checked using all the relevant tests specified in Annex A (normative) of this document. The framework, concepts, and methodology for testing, and the criteria to be achieved to claim conformance are specified in the OGC Compliance Testing Policies and Procedures and the OGC Compliance Testing web site.
 
-[#conf_class_uris,reftext='{table-caption} {counter:table-num}']
+
 .Conformance class URIs
 [cols="40,60",options="header"]
-!===
```

Comment:
Encode this as a conformance class. See https://www.metanorma.org/author/ogc/topics/requirements/

Reply:
@ronaldtse, I think this is not a Conformance Class per se: 

````
.Conformance class URIs
[cols="40,60",options="header"]
|===
| Conformance class | URI

|<<ats_core,Core>> |http://www.opengis.net/spec/ogcapi-processes-1/1.0/conf/core
|<<ats_ogc-process-description,OGC Process Description>> |http://www.opengis.net/spec/ogcapi-processes-1/1.0/conf/ogc-process-description
|<<ats_json,JSON>> |http://www.opengis.net/spec/ogcapi-processes-1/1.0/conf/json
|<<ats_html,HTML>> |http://www.opengis.net/spec/ogcapi-processes-1/1.0/conf/html
|<<ats_oas30,OpenAPI Specification 3.0>> |http://www.opengis.net/spec/ogcapi-processes-1/1.0/conf/oas30
|<<ats_job-list,Job list>> |http://www.opengis.net/spec/ogcapi-processes-1/1.0/conf/job-list
|<<ats_callback,Callback>> |http://www.opengis.net/spec/ogcapi-processes-1/1.0/conf/callback
|<<ats_dismiss,Dismiss>> |http://www.opengis.net/spec/ogcapi-processes-1/1.0/conf/dismiss
|===
````

It doesn't have a `label`, a `subject`, nor a `classification` like the other ones.
From my point of view, it is just a table that lists a bunch of URIs.

Reply:
AH yes, you're right. Thanks.